plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditorBase.java

```diff
@@ -1309,6 +1318,12 @@ protected boolean isNavigationTarget(Annotation annotation) {
         return super.isNavigationTarget(annotation);
     }
 
+    @Nullable
+    protected DBPDataSourceContainer getDataSourceContainerForSyntaxRuleReloading() {
```

Comment:
```suggestion
    protected DBPDataSourceContainer getDataSourceContainer() {
```
Is it possible to use this container for something else?

Reply:
It is possible, but we got a bug with a simple 'get' name and decided to name it differently so it looks safe but awkward; I agree